docs/source/troubleshooting/troubleshooting-getting-started.adoc

```diff
@@ -138,3 +138,14 @@ Downloading https://github.com/minishift/minishift/releases/download/v1.2.0/mini
  65 B / 65 B [===================================================================================================================================================] 100.00% 0s
 Update failed: open /usr/bin/.minishift.new: permission denied
 ----
+
+[[safari-openshift-web-console-issue]]
+== Safari browser issue with OpenShift web console
+
+OpenShift web console URL i.e. `minishift console` does not work on older versions of Safari web browser e.g. version 10.1.2 (12603.3.8) and its gives following error:
```

Comment:
The use of i.e. and e.g. here confuses the sentence rather than clarifying it. We can try rewriting it with something along these lines:

```adoc
The OpenShift web console does not work on older versions of the Safari web browser such as version 10.1.2 (12603.3.8).
Attempting to access the web console results in the following error:
```